### PR TITLE
create the documentation of card component

### PR DESCRIPTION
**Pull request by Mady-Camara**

### CODE REVIEW

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor [hunk 1]
```diff
@@ -1,6 +1,170 @@
...
+        <h2>Card</h2>
+        <p>Cards are a popular and flexible UI component.</p>
+        <hr />
+    </SectionLeft>
+</SectionWrapper>
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Basics</h2>
+        <p>
+            A card is just an element with a <Code Content="PENhcmQ+"></Code> applied. You can put any kind of content inside. Make sure you wrap your content in a <Code Content="PENhcmRTZWN0aW9uPg=="></Code> element in order to achieve the traditional card look.
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor:14]: A card is just an element with a <Card> applied .... its just a copy of the statement based on the fact that class is applied that doesn't make any sense for component... you can even remove this sentence!... element refers to html we are using components not elements.

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor [hunk 2]
```diff
@@ -1,6 +1,170 @@
...
+        </p>
+        <p>
+            A card container has no padding, allowing you to place full-bleed images inside. Use the <Code Content="PENhcmREaXZpZGVyPg=="></Code> and <Code Content="PENhcmRTZWN0aW9uPg=="></Code> tags to sub-divide a card.
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor:17]: Tags is Component

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor [hunk 3]
```diff
@@ -1,6 +1,170 @@
...
+            <OpenCodeDev.Blazor.Foundation.Components.Containers.CardDivider>
+                This is a header
+            </OpenCodeDev.Blazor.Foundation.Components.Containers.CardDivider>
+            <img src="https://images.hdqwalls.com/download/saturn-planet-8k-3i-1336x768.jpg">
+            <OpenCodeDev.Blazor.Foundation.Components.Containers.CardSection>
+                <h4>This is a card.</h4>
+                <p>It has an easy to override visual style, and is appropriately subdued.</p>
+            </OpenCodeDev.Blazor.Foundation.Components.Containers.CardSection>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.Card>
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout TypeClass="alert">
```
Comment (maxgsamson) [inline Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor:31]: We do not support IE so notice must be removed

**Review by maxgsamson — CHANGES_REQUESTED** [review]

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor (hunk 1 above)
Comment (Mady-Camara) [reply Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor:14]: I deleted the first sentence and kept the rest
```suggestion
              You can put any kind of content inside. Make sure you wrap your content in a <Code Content="PENhcmRTZWN0aW9uPg=="></Code> Component in order to achieve the traditional card look.
```

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor (hunk 2 above)
Comment (Mady-Camara) [reply Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor:17]: I changed tags to components
```suggestion
            A card container has no padding, allowing you to place full-bleed images inside. Use the <Code Content="PENhcmREaXZpZGVyPg=="></Code> and <Code Content="PENhcmRTZWN0aW9uPg=="></Code> components to sub-divide a card.
```

File: Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor (hunk 3 above)
Comment (Mady-Camara) [reply Blazor_Foundation_6_Test_Shared/Documentation/Containers/Card.razor:31]: I deleted a Callout
```suggestion
        <OpenCodeDev.Blazor.Foundation.Components.Containers.CardSection>
                <h4>This is a card.</h4>
                <p>It has an easy to override visual style, and is appropriately subdued.</p>
            </OpenCodeDev.Blazor.Foundation.Components.Containers.CardSection>
        </OpenCodeDev.Blazor.Foundation.Components.Containers.Card>
        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
```

**Review by maxgsamson — APPROVED** [review]

**Pull request merged**